jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -68,16 +70,16 @@ public static JarType determineJarType(Path jarPath) throws IOException {
   }
 
   /**
-   * Explodes jar and create three layers for classes, resources and dependencies on container.
+   * Creates three layers for classes, resources, and dependencies on container for a standard jar.
```

Comment:
Sorry, this was my fault but how about 
```suggestion
   * Creates layers for dependencies, snapshot dependencies, resources and class on container for a standard jar.
```